src/main/java/com/exadel/discount/entity/User.java

```diff
@@ -0,0 +1,47 @@
+package com.exadel.discount.entity;
+
+import com.exadel.discount.config.EnumPostgresSQLType;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.UUID;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Entity
+@Table(name = "users")
+@TypeDef(
+        name = "user_role",
+        typeClass = EnumPostgresSQLType.class
+)
+public class User {
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(
+            name = "UUID",
+            strategy = "org.hibernate.id.UUIDGenerator"
+    )
+    private UUID id;
+
+    private String email;
+
+    private String password;
```

Comment:
It should be required I believe

Reply:
This was resolved in the commit "042a43c".